velox/exec/GroupingSet.h

```diff
@@ -43,6 +43,13 @@ class GroupingSet {
 
   ~GroupingSet();
 
+  // GroupingSet is also used for MarkDistinctOperator. When being used so,
```

Comment:
Perhaps, shorten to "Used by MarkDistinct operator to identify rows with unique values."